.checkpatch.conf

```diff
@@ -8,4 +8,12 @@
 --ignore=PREFER_KERNEL_TYPES
 --ignore=CONCATENATED_STRING
 --ignore=USLEEP_RANGE
+--ignore=COMPARISON_TO_NULL
```

Comment:
No. Even if existing code has many non-compliant lines, we don't want new patches to ignore this.